<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoundResultWidget</class>
 <widget class="QWidget" name="RoundResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="playerName">
     <property name="text">
      <string>Player Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="winSpinBox">
     <property name="suffix">
      <string> wins</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="confirmedIndicator">
     <property name="text">
      <string>Confirmed</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
